tests/Unit/Core/Form/IdentifiableObject/CommandBuilder/CommandBuilderTest.php

```diff
@@ -365,12 +365,12 @@ public function getMultiShopCommandsParameters(): iterable
         // Same test but now url is a multishop field
         $config = new CommandBuilderConfig(self::MULTI_SHOP_PREFIX);
         $config
-            ->addMultiShopField('[url]', 'setUrl', CommandField::TYPE_STRING)
-            ->addMultiShopField('[name]', 'setName', CommandField::TYPE_STRING)
-            ->addMultiShopField('[command][isValid]', 'setIsValid', CommandField::TYPE_BOOL)
-            ->addMultiShopField('[_number]', 'setCount', CommandField::TYPE_INT)
-            ->addMultiShopField('[parent][children]', 'setChildren', CommandField::TYPE_ARRAY)
-            ->addField('[date_time]', 'setDate', CommandField::TYPE_DATETIME)
+            ->addMultiShopField('[url]', 'setUrl', DataField::TYPE_STRING)
+            ->addMultiShopField('[name]', 'setName', DataField::TYPE_STRING)
+            ->addMultiShopField('[command][isValid]', 'setIsValid', DataField::TYPE_BOOL)
+            ->addMultiShopField('[_number]', 'setCount', DataField::TYPE_INT)
+            ->addMultiShopField('[parent][children]', 'setChildren', DataField::TYPE_ARRAY)
+            ->addField('[date_time]', 'setDate', DataField::TYPE_DATETIME)
```

Comment:
You should maybe add a test case in this Unit test class about the compound fields (you will have to update the `CommandBuilderTestCommand` accordingly but it's worth testing the whole chain)